README.md

```diff
@@ -172,8 +172,9 @@ Compatibility with Linux distributions based on Debian, Ubuntu or Redhat familie
 
 The continuous integration setup of this project currently covers following platforms:
 - CentOS 6, 7 and 8
-- Debian 8, 9 and 10
-- Ubuntu 14.04, 16.04, 18.04 and 20.04
+- Rocky Linux 8
```

Comment:
Looking at the matrix, we do not really test with all those platforms, do we? I can only see:

```
          - centos
           - debian
           - ubuntu
```

So we should either update the matrix or update documentation. @lpaulmp what do you think?

Reply:
I have committed a faux pas by adding these in OSs into this PR.  I have reasons for doing so, but by me slowing us down to talk about this in this PR, I'm only serving to delay getting a running CI workflow going, which is certainly more important.

I'm going to carve these out of this PR, and propose them in a future PR where we can discuss which OSs are appropriate.